plugins/CoreAdminHome/Emails/TokenAuthDeletedEmail.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins\CoreAdminHome\Emails;
+
+use Piwik\Piwik;
+use Piwik\Plugins\CoreAdminHome\Emails\SecurityNotificationEmail;
+
+class TokenAuthDeletedEmail extends SecurityNotificationEmail
+{
+    protected function getBody()
+    {
+        return Piwik::translate('CoreAdminHome_SecurityNotificationTokenAuthDeletedBody');
+    }
+}
```

Comment:
Would it help to include the name of the deleted token auth here so the user knows which one they have to recreate?